From cc2c072c788fb03fce4f6b54079c5d16a535e8fe Mon Sep 17 00:00:00 2001
From: Anurag Chouhan <achouhan@codeaurora.org>
Date: Wed, 5 Jul 2017 14:34:36 +0530
Subject: [PATCH 1/3] Wcnss_service: Add build support for full treble

Add support to build wcnss_service and copy to /vendor/bin/
path for full treble.

Change-Id: Ib014e632599ffab6a7d061d6c2de5687487e5435
CRs-fixed: 2071416
Signed-off-by: sub77 <sub77@ymail.com>
---
 qcwcn/wcnss-service/Android.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/qcwcn/wcnss-service/Android.mk b/qcwcn/wcnss-service/Android.mk
index 6662457..32bc507 100644
--- a/qcwcn/wcnss-service/Android.mk
+++ b/qcwcn/wcnss-service/Android.mk
@@ -2,6 +2,9 @@
 ifneq (,$(filter arm aarch64 arm64, $(TARGET_ARCH)))
 LOCAL_PATH := $(call my-dir)
 include $(CLEAR_VARS)
+ifeq ($(PRODUCT_VENDOR_MOVE_ENABLED),true)
+LOCAL_PROPRIETARY_MODULE := true
+endif
 LOCAL_MODULE := wcnss_service
 LOCAL_C_INCLUDES += $(TARGET_OUT_HEADERS)/common/inc/
 LOCAL_SRC_FILES := wcnss_service.c
-- 
2.1.2

